Add x9 service bit support to dnsseed.bluematt.me

TheBlueMatt opened the pull request.

TheBlueMatt committed:
Comment that most dnsseeds only support some service bits combos

TheBlueMatt committed:
Add x9 service bit support to dnsseed.bluematt.me

MarcoFalke commented:
ACK e5f8b0d

MarcoFalke commented on src/chainparams.cpp:
```diff
@@ -113,12 +113,13 @@ class CMainParams : public CChainParams {
-        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be", true)); // Pieter Wuille
-        vSeeds.push_back(CDNSSeedData("bluematt.me", "dnsseed.bluematt.me")); // Matt Corallo
+        // Note that of those with the service bits flag, most only support a subset of possible options
+        vSeeds.push_back(CDNSSeedData("bitcoin.sipa.be", "seed.bitcoin.sipa.be", true)); // Pieter Wuille, only supports x1, x5, x9, and x13
```
Are you sure @sipa supports x13. (Right now I don't get an answer)

laanwj commented:
Can we merge "Marking filter support for cdecker's DNS seed #8941" into this? Would make sense to do this update at the same time.

cdecker committed:
My DNS seed supports filtering

Currently supports filtering by any combination of the 4 least
significant bits.

TheBlueMatt commented:
Included #8941.

cdecker commented:
Thanks Matt, closing #8941

laanwj commented:
utACK 2449e12

MarcoFalke commented:
utACK 2449e12

btcdrak commented:
utACK 2449e12681ab549695f143d9b3ad2c24be03a78d

sipa commented:
utACK

jonasschnelli commented:
post merge ACK
